### PR TITLE
Add boosted.finance

### PR DESCRIPTION
**Pull request by JoaquinLiHua**
Add new space for boosted.finance

### CODE REVIEW

File: spaces/boostedFinance/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "key": "boostedFinance",
```
Comment (e-asia) [inline spaces/boostedFinance/index.json:2]: Key should not contain uppercase 
```suggestion
  "key": "boosted-finance",
```

File: spaces/boostedFinance/skin.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.name {
```
Comment (e-asia) [inline spaces/boostedFinance/skin.scss:1]: Css selector should be similar to key
```suggestion
.boosted-finance {
```

**Review by e-asia — COMMENTED** [review]
Images should be 256x256 and lightweight

File: spaces/boostedFinance/skin.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.boosted-finance {
+  --primary-color: #9AE6B4;
```
Comment (bonustrack) [inline spaces/boostedFinance/skin.scss:2]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme. 
image

**Comment by bonustrack** [comment]
The folder name of your space must be exactly the same as your space key `boosted-finance`.

**Comment by JoaquinLiHua** [comment]
Fixed, thanks! Also, who do we contact for qudratic voting implementation? =)

**Review by e-asia — APPROVED** [review]
@JoaquinLiHua Maybe if you could just reduce image size before a merge? Thank you

**Comment by JoaquinLiHua** [comment]
Updated!

**Pull request merged**